scripts/types.d/chain_configs.ts

```diff
@@ -1,3 +1,3 @@
-import {readArtifact} from "../helpers.js";
+import { readArtifact } from "../helpers.js";
 
-export const chainConfigs: Config = readArtifact(`${process.env.CHAIN_ID}`, 'chain_configs');
\ No newline at end of file
+export const chainConfigs: Config = readArtifact(`${process.env.CHAIN_ID || "localterra"}`, 'chain_configs');
```

Comment:
We can set a chain ID and a user wallet in environment config file for Localterra same as for other networks. Maybe remove this [line](https://github.com/astroport-fi/astroport-core/blob/6d96537b2893d23ce848193698186876bb552cb1/scripts/helpers.ts#L67) and use [this](https://github.com/astroport-fi/astroport-core/blob/6d96537b2893d23ce848193698186876bb552cb1/scripts/helpers.ts#L64) for all networks

Reply:
It's even simpler not to specify anything for LocalTerra